packages/shared/consoleWithStackDev.js

```diff
@@ -15,13 +22,17 @@ import ReactSharedInternals from 'shared/ReactSharedInternals';
 
 export function warn(format, ...args) {
   if (__DEV__) {
-    printWarning('warn', format, args);
+    if (!suppressWarning) {
```

Comment:
Is this what prevents us from showing double warnings/errors in React Native's log box?

Reply:
This prevents us from showing double warnings/errors in React tests.